src/components/Timeline.js

```diff
@@ -2,9 +2,24 @@ import React from 'react';
 import './Timeline.css';
 import TimelineEvent from './TimelineEvent';
 
-const Timeline = () => {
+
+const Timeline = (props) => {
+  const eventsComponent = props.events.map((event) => {
+    return (
+      <TimelineEvent
+        person={event.person}
+        status={event.status}
+        timeStamp={event.timeStamp}
+        key={event.timeStamp}
+      />
+    );
+  });
   
-  return;
+  return (
+    <ul className="Timeline">
```

Comment:
Check the CSS files for the class names.

```suggestion
    <ul className="timeline">
```